Admin/Extension/PublishTimePeriodExtension.php

```diff
@@ -45,15 +51,27 @@ public function configureFormFields(FormMapper $formMapper)
             'required' => false,
         );
 
-        $formMapper->with($this->formGroup, array(
-            'translation_domain' => 'CmfCoreBundle',
-            ))
-            ->add('publish_start_date', 'date', $dateOptions, array(
-                'help' => 'form.help_publish_start_date',
-            ))
-            ->add('publish_end_date', 'date', $dateOptions, array(
-                'help' => 'form.help_publish_end_date',
+        if ($formMapper->hasOpenTab()) {
```

Comment:
lets make a decision on the mailinglist about this ..
I think its ok to make the jump but we should discuss this to ensure we do not cause us bigger pain ..